cypress/integration/invoices/invoices.exportBatchVoucher.spec.js

```diff
@@ -8,13 +8,18 @@ import NewFund from '../../support/fragments/finance/funds/newFund';
 import Funds from '../../support/fragments/finance/funds/funds';
 import DateTools from '../../support/utils/dateTools';
 import FileManager from '../../support/utils/fileManager';
+import SettingsInvoices from '../../support/fragments/invoices/settingsInvoices';
 
 describe('ui-invoices-settings: Export batch voucher', () => {
   const invoice = { ...NewInvoice.defaultUiInvoice };
   const vendorPrimaryAddress = { ...VendorAddress.vendorAddress };
   const invoiceLine = { ...NewInvoiceLine.defaultUiInvoiceLine };
   const fund = { ...NewFund.defaultFund };
   const subtotalValue = 100;
+  const batchGroupConfiguration = {
+    batchGroupId: 'cd592659-77aa-4eb3-ac34-c9a4657bb20f',
```

Comment:
Seems that is hardcode. It is possible to genereate it with uuid()?

Reply:
Done